uwblueprint/starter-code-cli
# Fix yarn prod and add GitHub Action to auto publish

## Notion ticket link



[Fix yarn run prod script in CLI & build/deployment enhancements](https://www.notion.so/uwblueprintexecs/Fix-yarn-run-prod-script-in-CLI-build-deployment-enhancements-c22136502d4446779d69a2d09018ee99)



## Implementation description

- Enables Json modules for `tsc`
- Adds GitHub Action to publish to npm on push to `main` if the version number changes

Sample run:
https://github.com/uwblueprint/starter-code-cli/runs/2651473755
![image](https://user-images.githubusercontent.com/18404076/119277229-6ec4cf80-bbec-11eb-82a2-87875a093f7b.png)




## Steps to test

1. Test `yarn prod` locally and check if built files from yarn prod is working as intended
2. Change version number before merging the PR to make sure an actual deployment works?

## Checklist
